src/cart/crt_context.c

```diff
@@ -990,49 +993,48 @@ crt_context_req_untrack(struct crt_rpc_priv *rpc_priv)
 	credits = crt_gdata.cg_credit_ep_ctx - inflight;
 	while (credits > 0 && !d_list_empty(&epi->epi_req_waitq)) {
 		D_ASSERT(epi->epi_req_wait_num > 0);
-		rpc_priv = d_list_entry(epi->epi_req_waitq.next,
+		tmp_rpc = d_list_entry(epi->epi_req_waitq.next,
 					struct crt_rpc_priv, crp_epi_link);
-		rpc_priv->crp_state = RPC_STATE_INITED;
-		rpc_priv->crp_timeout_ts = crt_get_timeout(rpc_priv);
+		tmp_rpc->crp_state = RPC_STATE_INITED;
+		tmp_rpc->crp_timeout_ts = crt_get_timeout(tmp_rpc);
```

Comment:
Not a regression, but this line looks odd, crt_get_timeout() is only called from four places, and it's return value is always assigned to rpc->crp_timeout_ts, so it would seem that this could be replaced with a function called crt_set_timeout() which would combine these two operations and would make the code clearer.

Reply:
Agreed, will change.